packages/@sanity/default-login/config.dist.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What happens if non-enterprise users adds providers here?

Reply:
They will be sent to whatever URL they give, however they will not be able to produce an actual Sanity session.

Reply:
Maybe we should warn or give some kind of feedback if that happens?

Reply:
Yes, we now warn when the custom provider logo is clicked and the project is missing the feature. In this way someone can implement this before they get the actual bit.